Fix for issue with synchronizing payment method with Wallee when some of payment was deleted on Wallee side.

diff --git a/Model/PaymentMethodConfigurationManagement.php b/Model/PaymentMethodConfigurationManagement.php
index 3d1fbda..a39009c 100644
--- a/Model/PaymentMethodConfigurationManagement.php
+++ b/Model/PaymentMethodConfigurationManagement.php
@@ -120,7 +120,10 @@ class PaymentMethodConfigurationManagement implements PaymentMethodConfiguration
             $this->searchCriteriaBuilder->addFilter(PaymentMethodConfigurationInterface::STATE,
                 [
                     PaymentMethodConfiguration::STATE_ACTIVE,
-                    PaymentMethodConfiguration::STATE_INACTIVE
+                    PaymentMethodConfiguration::STATE_INACTIVE,
+                    // BEGINNING OF PATCH
+                    PaymentMethodConfiguration::STATE_HIDDEN
+                    // END OF PATCH
                 ], 'in')->create())
             ->getItems();
         foreach ($existingConfigurations as $existingConfiguration) {
